isotpsend.c

```diff
@@ -178,6 +182,22 @@ int main(int argc, char **argv)
 		    }
 		    break;
 
+	    case 'l':
+		    if (optarg[0] == 'i') {
+			    loops = 0; /* infinite loop */
+		    } else {
+			    loops = strtoul(optarg, (char **)NULL, 10);
+			    if (!loops) {
+				    fprintf(stderr, "Invalid argument for option -l !\n");
```

Comment:
nitpick: No space in front of the `!`?

Reply:
Yes. This is our common coding style in can-utils.

Reply:
Ah! The parser stopped at `if (!loops)` :-D

Will fix that.